src/routes/jira/atlassian-connect/jira-atlassian-connect-get.ts

```diff
@@ -29,7 +31,12 @@ const modules = {
 		name: {
 			value: "GitHub"
 		},
-		url: "https://github.com"
+		url: "https://github.com",
+		actions: {
+			createBranch: {
+				templateUrl: `${envVars.APP_URL}/github/create-branch?issue_key={issue.key}&issue_summary={issue.summary}`
```

Comment:
If required we can chuck `instance=${instance}` or `jira_host=${jiraHost}` here

Reply:
can we though?

Reply:
no :(